README.md

```diff
@@ -15,6 +15,8 @@ You can use built-in templates to deploy your game for the following common scen
 * Single-region fleet: Deploy your game server to one fleet in a single AWS Region. Use this scenario to experiment with your install scripts and runtime deployment, as well as your integration.
 * Spot fleet: Deploy your game server to a set of low-cost Spot fleets and a back-up On-Demand fleet. Use this scenario to experiment with a multi-fleet hosting structure that balances cost savings and durable game session availability.
 * FlexMatch fleet: Deploy your game server for hosting with a FlexMatch matchmaking solution. [Amazon GameLift FlexMatch](https://docs.aws.amazon.com/gamelift/latest/flexmatchguide/match-intro.html) is a highly scalable and customizable matchmaking service for multiplayer games. Use this scenario to set up basic matchmaking components (including a rule set) that you can customize.
+* Windows Dedicated Server Build Support module installed by going to Unity Hub -> Installs -> click on the gear button on the right of your editor -> Add modules

```

Comment:
I don't think this is in the right place. This should be under Prerequisites.

Also I don't think we need to explain how to install unity modules. We can edit the compatible unity editor item to be something like,
> A compatible Unity editor (...) with Dedicated Server modules for Windows (and Linux if desired).